example_config.yaml

```diff
@@ -13,6 +13,9 @@ chihaya:
   udp:
     addr: 0.0.0.0:6881
     allow_ip_spoofing: false
+    max_clock_skew: 10s
+    private_key: |
+      paste a random string here that will be used to hmac connection IDs
```

Comment:
yea we use it a lot for cloundconfigs: https://coreos.com/os/docs/latest/cloud-config.html#writefiles